library/spdm_common_lib/libspdm_com_crypto_service.c

```diff
@@ -575,34 +575,26 @@ uint8_t libspdm_get_cert_slot_count(libspdm_context_t *spdm_context)
 }
 
 /**
- * This function verifies peer certificate chain buffer including spdm_cert_chain_t header.
+ * This function verifies the integrity of peer certificate chain buffer including
+ * spdm_cert_chain_t header.
  *
  * @param  spdm_context                  A pointer to the SPDM context.
  * @param  cert_chain_buffer              Certificate chain buffer including spdm_cert_chain_t header.
  * @param  cert_chain_buffer_size          size in bytes of the certificate chain buffer.
- * @param  trust_anchor                  A buffer to hold the trust_anchor which is used to validate the peer certificate, if not NULL.
- * @param  trust_anchor_size             A buffer to hold the trust_anchor_size, if not NULL.
- * @param  is_requester                   Indicates if it is a requester message.
  *
- * @retval true  Peer certificate chain buffer verification passed.
- * @retval false Peer certificate chain buffer verification failed.
+ * @retval true  Peer certificate chain buffer integrity verification passed.
+ * @retval false Peer certificate chain buffer integrity verification failed.
  **/
-bool libspdm_verify_peer_cert_chain_buffer(libspdm_context_t *spdm_context,
-                                           const void *cert_chain_buffer,
-                                           size_t cert_chain_buffer_size,
-                                           const void **trust_anchor,
-                                           size_t *trust_anchor_size,
-                                           bool is_requester)
+bool libspdm_verify_peer_cert_chain_buffer_integrity(libspdm_context_t *spdm_context,
+                                                     const void *cert_chain_buffer,
+                                                     size_t cert_chain_buffer_size)
 {
-    const uint8_t *root_cert;
-    size_t root_cert_size;
-    uint8_t root_cert_hash[LIBSPDM_MAX_HASH_SIZE];
-    size_t root_cert_hash_size;
-    const uint8_t *received_root_cert;
-    size_t received_root_cert_size;
     bool result;
-    uint8_t root_cert_index;
     bool is_device_cert_model;
+    bool is_requester;
+
+    /*peer is_requester is opposite of local*/
+    is_requester = !spdm_context->local_context.is_requester;
```

Comment:
`is_requester` always specifies the local entity. This is backwards.

Reply:
Thanks. I have changed the comment and made the `is_requeseter` to `is_peer_requester`.